utils/misc.py

```diff
@@ -0,0 +1,5 @@
+
+class AttrDict(dict):
```

Comment:
That AttrDict thing is amazing! Can we trash the whole Config class and use this instead? 👀

Reply:
Maybe depends if we have to add other methods to config